airqo-platform/AirQo-frontend
# Global state review

- What does this PR do?
This PR consumes the global state using Redux and displays the name of the logged in user on the dashboard and in the profile settings. The same PR also implements the Device Registry frontend component.
- What JIRA task is related to this PR?
No card was created for this. We shall need to smoothen the process of adding new "bug" cards during the sprint.
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate/` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test.
Please ensure that the latest auth-service is also running on localhost by running: `cd AirQo-api/auth-service` and then `npm install` and then `npm start`
Important to note that WINDOWS users will need to change the `scripts` property of` "local": "PORT=5000 react-scripts start"` inside **AirQo-frontend/locate/package.json**  to `"local": "react-scripts start"`
You are also supposed to create an **.env** file inside **AirQo-frontend/locate/**  directory and update it with contents that you will get from Martin either through Slack or email.
- Which end points should are ready for testing?
No end points available.
